### ISSUE TITLE
reader not scrolling : IOS 13.x

### ISSUE THREAD

**Issue by ronghester** [readium/SDKLauncher-iOS#103]
encountered a problem on IOS app made with Readium 1. The reader page is not scrolling on devices running IOS 13.x. When the build is generated with xcode 11

**Comment by danielweck** [comment]
I suspect this webkit-specific bug can (in principle) be corrected in https://github.com/readium/readium-shared-js

... in practice though, the legacy Readium1 Javascript codebase is not actively maintained, so I would not hold my breath for a rapid resolution here in GitHub by the open source community.

Regardless, I would suggest you should post further details about the rendering mode used (single page scroll, continuous scroll, paginated, two-page spread, etc.), screen orientation / size, etc. type of publication (mixed fixed-layout / reflowable), etc.

**Comment by ronghester** [comment]
This issue exist in the latest version of IOS SDK launcher as well. Steps to reproduce

1. Clone latest version of SDK launcher 
2. Build using xcode 11 and run on device running 13.x 
3. Open epub and user won't be able to scroll. 

Please let us know if the issue is valid. 

Thanks

**Comment by danielweck** [comment]
I wonder if this issue could be related to `document.scrollingElement` vs. `document.body`.
See: https://github.com/edrlab/webpub-viewer/issues/3

**Comment by FirstyGroup** [comment]
We've investigated this problem and found the following:

1. SDKLauncher-iOS compiles fine in the latest version of Xcode 11.1 (11A1027) and runs on the latest version of iOS (13.1.3)

2. There are no problems with the text scrolling, "Continuous" option has to be selected in the Settings -> Scroll

3. We were able to recreate the page swipe problem in the app based on the SDKLauncher-iOS code but the issue turned out to be unrelated to the SDKLauncher-iOS itself and was in fact a trivial bug in the code related to the Gesture Recognisers
